### PR TITLE
file added

### PR DESCRIPTION
**Pull request by AmitPaswan18**

### CODE REVIEW

File: src/components/TodoItems.jsx [hunk 1]
```diff
@@ -0,0 +1,378 @@
+import { useState, useEffect, useRef } from "react";
+import { RiDeleteBin5Fill, RiEditBoxLine } from "react-icons/ri";
+import Checkboxbutton from "./common/Checkboxbutton";
+import ToogleBar from "./common/ToogleBar";
+import {} from "react-icons/ri";
+import Button from "./common/button";
+import RadioButton from "./common/RadioButton";
+import InputButton from "./common/InputButton";
```
Comment (yuvraj-singh-codes) [inline src/components/TodoItems.jsx]: add space between import and function

File: src/components/TodoItems.jsx [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+    return true;
+  }
+
+  const deleteTodo = (index) => {
+    let updatedtask = [...todo];
+    updatedtask.splice(index, 1);
+    setTodo(updatedtask);
+    saveTasks([...updatedtask]);
+  };
+  function handleSubmit(event) {
```
Comment (yuvraj-singh-codes) [inline src/components/TodoItems.jsx]: always add space between handlers,

Comment (AmitPaswan18) [reply src/components/TodoItems.jsx]: ok sir

File: src/components/TodoItems.jsx [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+                                  placeholder="Task..."
+                                  onChange={(e) => setTitleEdit(e.target.value)}
+                                  onBlur={handleBlur}
+                                  value={titleEdit}
+                                  onKeyDown={(e) => {
+                                    if (e.key === "Enter") {
+                                      handleEditSubmit(e);
+                                    }
+                                  }}
+                                />
```
Comment (yuvraj-singh-codes) [inline src/components/TodoItems.jsx]: remove this onKeyDown and use form tag for this.
Rename the component name with a meaningful name.

File: src/components/TodoItems.jsx [hunk 4]
```diff
@@ -0,0 +1,378 @@
...
+                            />
+                            {checkedItems[index] == true ? (
+                              <span className="badge text-bg-success m-2">
+                                Complete
+                              </span>
+                            ) : null}
+                          </div>
+                        </div>
+                      )}
+                      <div className="flex flex-col md:flex-row">
```
Comment (yuvraj-singh-codes) [inline src/components/TodoItems.jsx]: do not push commented on code on git

**Pull request merged**